packages/ui/src/common/components/statistics/FractionValue.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+
+import { Colors } from '@/common/constants'
+
+import { NumericValue } from './NumericValueStat'
+
+export interface FractionValueProps {
+  numerator: ReactNode | number
+  denominator: ReactNode | number
+}
+
+export const FractionValue = ({ numerator, denominator }: FractionValueProps) => (
+  <NumericValue>
+    {numerator}
+    <DenominatorValue>/ {denominator}</DenominatorValue>
+  </NumericValue>
```

Comment:
I think that we have a similar component. I'm not sure if it can be re-used here, so if not then this one is OK.

Reply:
I looked for an existing component like in `WorkingGroupsOpening.tsx`: ![WorkingGroupsOpening](https://user-images.githubusercontent.com/6571453/123105394-11fd4480-d438-11eb-9723-98cf98d67cea.png), but it doesn't seem to be implemented yet. Did I miss an already existing component?

Reply:
No, that's OK than. No need to change this. I wanted to be sure if we can't re-use.